packages/dd-trace/src/profiling/profilers/wall.js

```diff
@@ -267,30 +290,16 @@ class NativeWallProfiler {
     return labels
   }
 
-  profile () {
-    return this._stop(true)
+  profile ({ restart = true } = {}) {
```

Comment:
I like `{ restart = true} = {}`. This is fun :-) OTOH, we don't use `= {}` in the events profiler, and I'm unclear on where would we call this method without arguments. Is it from tests?